open_lm/main.py

```diff
@@ -81,7 +81,7 @@ def natural_key(string_):
 def get_latest_checkpoint(path: str):
     is_s3 = path.startswith("s3")
     fs, root_path = fsspec.core.url_to_fs(path)
-    checkpoints = fs.glob(os.path.join(root_path, "**/epoch_*.pt"))
+    checkpoints = fs.glob(os.path.join(root_path, "epoch_*.pt"))
```

Comment:
is root_path here what is referred to as checkpoint_path elsewhere?

Reply:
yup!